components/style/values/generics/length.rs

```diff
@@ -207,6 +207,15 @@ impl<LengthPercentage> MaxSize<LengthPercentage> {
     pub fn none() -> Self {
         MaxSize::None
     }
+
+    /// Convert
+    #[cfg(not(feature = "gecko"))]
+    pub fn to_option(self) -> Option<LengthPercentage> {
```

Comment:
Nit: Just name that `length`, like `Result::ok`?

Reply:
This is subtle but I mean this method not as "if this enum is the `LengthPercentage` variant, access its field" but rather "losslessly convert all the information in this enum". If/when we add support for e.g. `min-width: min-content` https://drafts.csswg.org/css-sizing/#min-size-properties this enum will have additional variants like it does today for Gecko, this method will need to be removed, and the callers should be modified to handle those variants separately.